Source/FlowEditor/Private/Graph/FlowGraphSchema.cpp

```diff
@@ -54,9 +54,9 @@ void UFlowGraphSchema::SubscribeToAssetChanges()
 	}
 }
 
-void UFlowGraphSchema::GetPaletteActions(FGraphActionMenuBuilder& ActionMenuBuilder, const UClass* AssetClass, const FString& CategoryName)
+void UFlowGraphSchema::GetPaletteActions(FGraphActionMenuBuilder& ActionMenuBuilder, const UFlowAsset* EditedFlowAsset, const FString& CategoryName)
```

Comment:
This is reworked to not use the default flow asset, but instead use the actual flow asset (that is being edited) so we can ask the right class about which nodes are legal for the palette.